capstone/src/main/java/com/google/sps/servlets/ClubServlet.java

```diff
@@ -1,13 +1,29 @@
 package com.google.sps.servlets;
```

Comment:
nit: could you add a separate directory for this and the corresponding test file